examples/build_twilio_go.py

```diff
@@ -34,6 +35,13 @@ def generate(openapi_spec_path, go_path, domain, is_file=False, language='go'):
               f"{go_path}/{sub_dir}/{domain_name}/{api_version}"
     os.system(command)
 
+    # Find files that only have read-only operations
+    files = glob.glob(f"{go_path}/{sub_dir}/**/*.*", recursive=True)
```

Comment:
Will this also remove all read only operations for twilio-go or just for terraform?

Reply:
Yes! Great catch, fixed.